lib/context.js

```diff
@@ -1043,17 +1043,29 @@ function _expandIri(activeCtx, value, relativeTo, localCtx, defined, options) {
 
   // prepend vocab
   if(relativeTo.vocab && '@vocab' in activeCtx) {
-    return activeCtx['@vocab'] + value;
+    value = activeCtx['@vocab'] + value;
   }
 
   // prepend base
-  if(relativeTo.base && '@base' in activeCtx) {
+  else if(relativeTo.base && '@base' in activeCtx) {
```

Comment:
This won't let me suggest here, but the style needs to be cleaned up to be something like this:

```js
    value = activeCtx['@vocab'] + value;
  } else if(relativeTo.base && '@base' in activeCtx) {
    // prepend base
```

Reply:
Updated locally